ui/src/components/TrafficPage.tsx

```diff
@@ -64,6 +64,7 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({setAnalyzeStatus, onTLS
     const [disableScrollList, setDisableScrollList] = useState(false);
 
     const ws = useRef(null);
+    const wsCli = useRef(null);
```

Comment:
this can be removed.

Reply:
@gadotroee oops, I'm sorry. I forgot to remove that line. Removed with https://github.com/up9inc/mizu/pull/304/commits/9428fa55ceab70e63b5883133899210d51cb69e0